xAPI.md

```diff
@@ -2114,56 +2099,47 @@ The time at which the experience occurred.
+the time of when the events described within this Statement occurred. If it is not included in the Statement when it is 
+submitted to the LRS, the LRS populates it with the same value it would use with [Stored](#stored).
 
-A timestamp in a Statement can differ from 
-[Stored](#stored) (the time at which the statement is stored). Namely, there can be delays between the occurrence of the 
-experience and the reception of the corresponding Statement by the LRS. 
+The timestamp property in a Statement can differ from the [stored property](#stored) (the time at which the statement is 
+stored). Namely, there can be delays between the occurrence of the experience and the reception of the corresponding 
+Statement by the LRS. 
 
-Where the experience occurs over a period of time, the timestamp can represent the start, end or any point of time 
+Where the experience occurs over a period of time, the timestamp property can represent the start, end or any point of time 
 during the experience. It is expected that communities of practice will define an appropriate point to record the 
-timestamp for different experiences. For example when recording the experience of eating at a restaurant, it might 
-be most appropriate to record the timestamp of the start of the experience; when recording the experience of 
-completing a qualification, it might be most appropriate to record the timestamp of the end of the experience.
+Timestamp for different experiences. For example when recording the experience of eating at a restaurant, it might 
+be most appropriate to record the Timestamp of the start of the experience; when recording the experience of 
+completing a qualification, it might be most appropriate to record the Timestamp of the end of the experience.
 These examples are for illustrative purposes only and are not meant to be prescriptive.
 
 ###### Requirements
-* A timestamp MUST be formatted according to [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations).
-* The timestamp property SHOULD* be set by the LRS to the value of [Stored](#stored) if not provided.
-* A timestamp SHOULD* include the time zone.
-* If the timestamp includes a time zone, the LRS MAY be return the timestamp using a different timezone to the one originally used in the statement
-so long as the point in time referenced is not affected. The LRS SHOULD* return the timestamp in UTC timezone. 
-* A timestamp MAY represent any point during an experience, not necessarily the beginning or end. 
-* A timestamp MAY be truncated or rounded to a precision of at least 3 decimal digits for seconds (millisecond precision MUST be preserved). 
-* An AP MUST NOT use a future time for a timestamp in a Statement.
-* An LRS MUST NOT reject a timestamp for being from the future, to prevent issues due to clock errors.
-* A SubStatement MAY have a timestamp that is in the future.
+
+* For requirements pertaining to the Timestamp data type, click [here](#timestamps).
```

Comment:
Instead of "click [here]", can we have "see [section X]" - I know this adds maintenance, but I also know that people print the doc or use the pdf version.